tests/wen_new_standard.test.ts

```diff
@@ -0,0 +1,687 @@
+
+  const [manager] = anchor.web3.PublicKey.findProgramAddressSync(
+    [MANAGER_SEED],
+    program.programId
+  );
+
+  before(async () => {
+    await program.methods.initManagerAccount().rpc();
+  });
+
+  describe("manager", () => {
+    describe("after initializing", () => {
+      let account: AccountInfo<Buffer>;
+
+      before(async () => {
+        account = await program.account.manager.getAccountInfo(manager);
+      });
+
+      it("should exist with a fixed seed", async () => {
+        expect(account.data).to.eql(
+          Buffer.from([221, 78, 171, 233, 213, 142, 113, 56])
+        );
+      });
+
+      it("should be owned by the program", async () => {
+        expect(account.owner).to.eql(program.programId);
+      });
+    });
+  });
+
+  describe("mint", () => {
+    const mintKeyPair = Keypair.generate();
+    const receiver = Keypair.generate();
+
+    const mintAuthPublicKey = provider.wallet.publicKey;
+    const mintPublicKey = mintKeyPair.publicKey;
+    const mintTokenAccount = getAssociatedTokenAddressSync(
+      mintPublicKey,
+      mintAuthPublicKey,
+      false,
+      TOKEN_2022_PROGRAM_ID
+    );
+
+    const receiverTokenAccount = getAssociatedTokenAddressSync(
+      mintPublicKey,
+      receiver.publicKey,
+      false,
+      TOKEN_2022_PROGRAM_ID
+    );
+
+    describe("after creating", () => {
+      let mintAccount: Mint;
+      let mintAccountInfo: AccountInfo<Buffer>;
+      let metadataPointer: Partial<MetadataPointer> | null;
+
+      before(async () => {
+        await program.methods
+          .createMintAccount({
+            permanentDelegate: null,
+            name: faker.lorem.text(),
+            symbol: faker.lorem.word(),
+            uri: faker.internet.url(),
+          })
+          .accounts({
+            authority: mintAuthPublicKey,
+            mint: mintPublicKey,
+            mintTokenAccount,
+            payer: mintAuthPublicKey,
+            receiver: mintAuthPublicKey,
+          })
+          .signers([mintKeyPair])
+          .rpc({
+            skipPreflight: true,
+            preflightCommitment: "confirmed",
+            commitment: "confirmed",
+          });
+
+        // getAccountInfo
+        mintAccountInfo = await provider.connection.getAccountInfo(
+          mintPublicKey
+        );
+        mintAccount = await getMint(
+          provider.connection,
+          mintPublicKey,
+          "confirmed",
+          TOKEN_2022_PROGRAM_ID
+        );
+        metadataPointer = getMetadataPointerState(mintAccount);
+      });
+
+      it("should be owned by the token extensions program", async () => {
+        expect(mintAccountInfo.owner).to.eql(TOKEN_2022_PROGRAM_ID);
+      });
+
+      it("should have metadata pointer", async () => {
+        expect(metadataPointer.metadataAddress).to.eql(mintPublicKey);
+      });
+    });
+
+    describe("after updating", () => {
+      before(async () => {});
+      it.skip("should have updated title", async () => {});
```

Comment:
Do some review into why this can not be set. Is this token extensions enforcing the no update.

Reply:
Token22 allows you to update a "Field" in additional metadata, but not the first fields "name", "symbol", "URI" @balmy-gazebo.

Reply:
Blame's on me. I was misunderstanding that you can't update the primary fields. Will update on these.